frontend/components/UserMenu/index.jsx

```diff
@@ -76,6 +77,14 @@ const UserMenu = (props) => {
         </List>
       </Portal>
       <Portal name={portals.NAV_MENU_ADDRESS_BOOK_AFTER} props={props} />
+
+      {/* Hide duplicated bottom header before logout menu entry (ios)*/}
+      <Helmet
```

Comment:
I've asked in the team about this and the response was, that it's not a good idea to misuse headers and especially the test id for this purpose. The test id could change and then this functionality would break and it would be hard to see, why it breaks, since it's a side effect.

Adding an empty portal is actually how this kind of functionality was planned to work.